### PR TITLE
fix: add image fix

### LINKED ISSUE #600: Images flipped 90 degrees
### Issue description

Images taken horizontally show up fine, but anything taken vertically is rendered horizontally.
Screenshot 2023-07-12 at 3 57 40 PM

The code is simply:
```
![Alt text](ghat3.jpg "Image caption")
![Alt text](ghat4.jpg "Image caption")
```
I feel like this can't be standard behavior, so maybe I accidentally changed a configuration file without realizing it. Otherwise, it might have to do with the way the theme tries to make all the images the same width.

### Theme version

v2.6.1

### Hugo version

hugo v0.112.1+extended

### Which browser rendering engines are you seeing the problem on?

Chromium

### PR DESCRIPTION
**Pull request by Jh123x**
- [x] fix: fix img flip
- [ ] test: test with image (No image to test)

Please let me know if anyone has an image to test

Closes #600

### CODE REVIEW

**Comment by stereobooster** [comment]
I think, it would be nice to have at least one such image in the repo to verify it's fixed (and verify in the future that it is not broken by any consequent change)

File: layouts/partials/picture.html [hunk 1]
```diff
@@ -81,6 +81,12 @@
         type="image/webp" 
       />
     {{ end }}
+    {{ with $img }}
```
Comment (stereobooster) [inline layouts/partials/picture.html]: there is `{{ with $img }}` on top, so this one is redundant

**Comment by stereobooster** [comment]
You can do something like this, to reduce repetition

```hugo
    {{ $s := newScratch }}
    {{ $s.SetInMap "sizes" "330w" 330 }}
    {{ $s.SetInMap "sizes" "660w" 660 }}
    {{ if gt .Width 1024 }}
      {{ $s.SetInMap "sizes" "1024w" 1024 }}
    {{ end }}
    {{ if gt .Width 1320 }}
      {{ $s.SetInMap "sizes" "2x" 1320 }}
    {{ end }}
    {{ if le .Width 1320 }}
      {{ $s.SetInMap "sizes" (printf "%dw" .Width) .Width }}
    {{ end }}

...

          srcset="
            {{ range $key, $size := $s.Get "sizes" }}
              {{ ($img.Resize (printf "%dx webp %s" $size $rotation)).RelPermalink }} {{ $key }},
            {{ end }}"

          srcset="
            {{ range $key, $size := $s.Get "sizes" }}
              {{ ($img.Resize (printf "%dx %s" $size  $rotation)).RelPermalink }} {{ $key }},
            {{ end }}"
```

to reduce repetition

**Comment by Jh123x** [comment]
> You can do something like this, to reduce repetition
> 
> ```
>     {{ $s := newScratch }}
>     {{ $s.SetInMap "sizes" "330w" 330 }}
>     {{ $s.SetInMap "sizes" "660w" 660 }}
>     {{ if gt .Width 1024 }}
>       {{ $s.SetInMap "sizes" "1024w" 1024 }}
>     {{ end }}
>     {{ if gt .Width 1320 }}
>       {{ $s.SetInMap "sizes" "2x" 1320 }}
>     {{ end }}
>     {{ if le .Width 1320 }}
>       {{ $s.SetInMap "sizes" (printf "%dw" .Width) .Width }}
>     {{ end }}
> 
> ...
> 
>           srcset="
>             {{ range $key, $size := $s.Get "sizes" }}
>               {{ ($img.Resize (printf "%dx webp %s" $size $rotation)).RelPermalink }} {{ $key }},
>             {{ end }}"
> 
>           srcset="
>             {{ range $key, $size := $s.Get "sizes" }}
>               {{ ($img.Resize (printf "%dx %s" $size  $rotation)).RelPermalink }} {{ $key }},
>             {{ end }}"
> ```
> 
> to reduce repetition

Thanks for the suggestion. I am still new to hugo and have to read up on how this works first before implementing it ><

**Comment by jpanther** [comment]
I think an issue with this implementation is that it's only dealing with one orientation. Ideally this needs to handle all the possible values and rotate the image accordingly.

**Pull request closed without merging**